doc/guides/maintaining-V8.md

```diff
@@ -144,9 +144,10 @@ includes the following branches<sup>1</sup>:
 </table>
 
 
-The versions of V8 used in Node.js v4.x and v6.x have already been abandoned by
-upstream V8. However, Node.js needs to continue supporting these branches for
-many months (Current branches) or several years (LTS branches).
+The versions of V8 used in Node.js v4.x, v6.x, and 8.x have already been
```

Comment:
Does it make sense to list the versions and continually update?  Wondering if we could just use some wording like "The versions of V8 used in Node.js have most often already been abanded by upstream V8."